src/test/resources/ide/language/js/Constructs.js

```diff
@@ -0,0 +1,13 @@
+
+var some_<warning>constnt</warning> = "1";
+
+class Some<warning>Classs</warning> {
+
+    constructor(config) {
+        this.member_varibleleel = config; // TODO Check member names
```

Comment:
By the way, for some reason with last plugin in ultimate this typo works

Reply:
Probably need to debug :)

Reply:
Also doesn't work in Idea, so it's a feature :)